Dev v0.1

generationmake opened the pull request:
create version v0.1

generationmake committed:
added kicad project

generationmake committed:
added schematic pdf

generationmake commented:
Hi @aentinger
could you please review this design?
Until now only the schematic is finished. But I have included all parts I want in the design. Do you think something is missing?
Thank you

generationmake committed:
started layout, added THT parts

generationmake committed:
switched to THT parts

generationmake committed:
changed DS18B20 to TO-92, cleaned up layout, added labels

generationmake committed:
updated pdf schematic and added images

generationmake committed:
renamed image files

generationmake commented:
good morning @aentinger
the first draft of schematic and layout is finished.

aentinger commented:
Thank you, will take a look soonish. Currently swamped.

generationmake commented:
no big deal. just take the time you need.
I just wanted to produce something as a starting point for discussion.

aentinger commented:
Looking good 👍 Could not find any issues with it 🤖

aentinger commented:
What do you think ... A simple (alphanumeric) LCD could be a nice addition, displacing facts such as Node ID, packets sent, etc.

aentinger commented:
And a HEX-Switch for selecting the Node ID :wink:

generationmake committed:
updated schematic and layout, added dip switch for configuration, added measurement for CANVCC

generationmake committed:
updated schematic pdf and images

generationmake commented:
Hi @aentinger 
I updated schematic and layout. I added a dip switch to make configurations (i. e. Node ID).
I also added a voltage devider to measure the voltage at the CAN bus.

I don't know a suitable display. I think the best solution would be to add a display on the I2C bus using the connector for the I2C bus. What do you think?

aentinger commented:
Looking very good, maybe one small improvement suggestion: Replace the I2C header with a Qwicc connector as it can be directly connected to a Qwiic LCD display, e.g. [this one](https://www.sparkfun.com/products/16398).

generationmake committed:
switched CAN transceiver to MCP2562 because of DIP package

generationmake committed:
added Qwiic connector

generationmake commented:
Hi @aentinger 

the main goal was to make a full THT design.
But you are right, a qwiic connector would be a good improvement. So I added it.
Any more suggestions?
The next step would be to add a lot of labels ... maybe also the uavcan label.

aentinger commented:
>the main goal was to make a full THT design.

Sorry, this was not fully clear to me (although I was wondering over the lack of SMD components :wink:) Who would be the intended target of this design? Regardless, one SMD component should be fine, if easy self-assembly is the goal.
